www-apps/radicale/radicale-2.1.12.ebuild

```diff
@@ -0,0 +1,98 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
```

Comment:
Please update eBuild to EAPI=7 while bumping it.

Reply:
Done.